ovs/lib/storagerouter.py

```diff
@@ -878,6 +948,9 @@ def remove_storagedriver(storagedriver_guid):
         if ServiceManager.has_service(voldrv_service, client=client):
             ServiceManager.disable_service(voldrv_service, client=client)
             ServiceManager.stop_service(voldrv_service, client=client)
+        if ServiceManager.has_service(cephproxy_service, client=client):
+            ServiceManager.disable_service(cephproxy_service, client=client)
+            ServiceManager.stop_service(cephproxy_service, client=client)
         if ServiceManager.has_service(dtl_service, client=client):
             ServiceManager.disable_service(dtl_service, client=client)
             ServiceManager.stop_service(dtl_service, client=client)
```

Comment:
The below piece of code will have to be extended for the ceph proxy as well. The alba proxy is started below for a short moment of time to allow the volumedriver to clean out the backend. It should be more or less identical.